search/lib/cmr.js

```diff
@@ -49,14 +50,28 @@ async function findCollections (params = {}) {
   return response.data.feed.entry;
 }
 
+async function stacIdToCmrCollectionId (providerId, stacId) {
+  const cacheKey = `stacId_${stacId}`;
+  let collectionId = myCache.get(cacheKey);
+  if (collectionId) {
+    return collectionId;
+  }
+  const cmrParams = stacCollectionToCmrParams(providerId, stacId);
```

Comment:
Not related to this PR, but the `stacCollectionToCmrParams` does not return the version parameter correctly.

Reply:
It's Node shorthand:

```
  return {
    provider_id: providerId,
    short_name: shortName,
    version
  };
 ```

You can provide just the field name when constructing objects if the field name is the same as the variable. So:

```
const a = 'a'
const b = 'b'
const c = 'c'
obj = { a, b, c }
```

`obj` becomes:
```
{
   a: 'a',
   b: 'b',
   c: 'c'
}
```

Reply:
Thanks! That shorthand is confusing for me, as someone who has minimal experience with JS.